examples/chip-tool/templates/logging/EntryToText-src.zapt

```diff
@@ -81,4 +81,16 @@ char const * GeneratedCommandIdToText(chip::ClusterId cluster, chip::CommandId i
 {{/zcl_clusters}}
 		default: return "Unknown";
 	}
+}
+
+char const * DeviceTypeIdToText(chip::DeviceTypeId id) {
+	switch(id)
+	{
+{{#zcl_device_types}}
+{{#if class}}
```

Comment:
Why is this conditioned on having "class"?  Please add a comment `{{! Commment here }}`explaining why.

Reply:
I overlooked it, that's definitely not needed. Fixed.